docs/Upgrading.md

```diff
@@ -15,17 +15,18 @@ version of React Native:
 
 ## 1. Upgrade the `react-native` dependency
 
-Note the latest version of the `react-native` npm package from here:
+Note the latest version of the `react-native` npm package from here (or use `npm info react-native` to check):
 
 * https://www.npmjs.com/package/react-native
 
-Open your `package.json` file and update the version of `react-native` under `dependencies` to the
-latest version. Now, in a terminal run:
+Now install that version of `react-native` in your project with `npm install --save`. For example, to upgrade to the version `0.18`, in a terminal run:
 
 ```sh
-$ npm install
+$ npm install --save react-native@0.18
 ```
 
+_NOTE:_ Some developers may use `react-native -v` to check current version. That's not correct. This shows the version of `react-native-cli`, which is a global command tool, but not `react-native`.
```

Comment:
I wonder if a better approach here is to have the CLI `-v` flag output the following:

```
react-native-cli: 0.x.y
react-native: 0.a.b
```

And if we are not inside of a project directory:

```
react-native-cli: 0.x.y
react-native: n/a - not currently inside React Native project directory
```